docs/source/guides/repository.mdx

```diff
@@ -0,0 +1,213 @@
+# Create and manage a repository
+
+The Huggingface Hub is a collection of git repositories. [Git](https://git-scm.com/) is a tool vastly used in software
+development to easily version projects when working collaboratively. This guide will show you how to interact with the
+repositories on the Hub, especially:
+
+- Create and delete a repository.
+- Manage branches and tags. 
+- Rename your repository.
+- Update your repository visibility.
+- Manage a local copy of your repository.
+
+<Tip warning={true}>
+
+If you are used to software engineering, working with platforms such as Gitlab/Github/Bitbucket, your first instinct
+might be to use `git` CLI to clone your repo (`git clone`), commit changes (`git add, git commit`) and push them
+(`git push`). This is still valid when using the Huggingface Hub. However, software engineering and machine learning do
+not share the same requirements and workflows. The `huggingface_hub` library has been implemented to address most of the
+ML use cases in a more efficient way. You can read our [Git vs HTTP paradigm](../concepts/git_vs_http) explanation page
+for more details.
```

Comment:
Briefly clarify what we mean by saying we address ML use cases more efficiently.

```suggestion
If you are used to working with platforms such as Gitlab/Github/Bitbucket, your first instinct
might be to use `git` CLI to clone your repo (`git clone`), commit changes (`git add, git commit`) and push them
(`git push`). This is still valid when using the Hugging Face Hub. However, software engineering and machine learning do
not share the same requirements and workflows. In ML, it may be more efficient to use HTTP, so you don't have to maintain a local folder with massive model weights. You can read our [Git vs HTTP paradigm](../concepts/git_vs_http) explanation page
for more details.
```